### DISCUSSION TITLE
Please continue this great work

### DISCUSSION THREAD

**Discussion by gryphs** [GlowstoneMC/Glowstone#1140] [General]
I really appreciate the work you guys do. Although I'm not a developer, I know how well you guys are doing and how groundbreaking it is. Why does the project seem to be stagnant - the stable branch is stuck at version 1.12.2 =( I really hope you guys could continue!

Best wishes ;)

**Comment by mastercoms** [comment]
Hi, since 1.13, we have been restructuring Glowstone to support all the new data formats and such which the base game now uses for blocks, biomes, and more. We have been making steady progress towards these goals but given the difficulty of the work and the volunteer nature of the project, it makes it difficult to allocate the necessary time for focusing on a complex feature to completion. We have had some blog posts since our 1.12 releases which explain this. Also, we have $1000 in bounties for 1.13 tasks so it's not like it's for lack of incentive, we really want to finish this anyway but it's just the complexity of the work which makes it take a long time. I'll be talking about it in a new blog post soon to recap all the great progress we've made since the last blog post.

**Reply by gryphs** [reply]
Thanks for the explanation mastercoms, I see that projects like Glowstone are also doing a lot of work on upgrading to version 1.13+. It is amazing that Glowstone has maintained nice compatibility with Bukkit, which is why I think Glowstone is superior to similar projects. Deeply appreciate your work and that of all the contributors, hopes that everything goes well.
